aws/request/retryer.go

```diff
@@ -69,11 +72,27 @@ func isCodeExpiredCreds(code string) bool {
 	return ok
 }
 
+func isSerializationErrorRetryable(err error) bool {
+	if err == nil {
+		return false
+	}
+
+	if aerr, ok := err.(awserr.Error); ok {
+		fmt.Println(aerr)
+		return isCodeRetryable(aerr.Code())
+	}
+
+	return strings.Contains(err.Error(), "Connection reset")
```

Comment:
Can we take advantage of,  [net.OpError](https://golang.org/pkg/net/#OpError) and 
`operr.Err.Error() == syscall.ECONNRESET.Error()`